acrobat/scripts/scripts.js

```diff
@@ -184,8 +184,10 @@ function decoratePromotion() {
   const { default: lanaLogging } = await import('./dcLana.js');
 
   // Setup CSP
-  const { default: ContentSecurityPolicy } = await import('./contentSecurityPolicy/csp.js');
-  ContentSecurityPolicy();
+  if (window.location.pathname.indexOf('online') > 0) {
+    const { default: ContentSecurityPolicy } = await import('./contentSecurityPolicy/csp.js');
+    ContentSecurityPolicy();
+  }
```

Comment:
Same as above, can we just move this code into `frictionless.js`?

Reply:
no, this needs to be in `scripts.js` for expedience